bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -755,3 +769,28 @@ data:
     {{- end }}
   {{- end }}
 {{- end }}
+---
+{{- if .Values.sentinel.service.createMaster}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-kubectl-scripts" (include "common.names.fullname" .) }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data:
+  update-master-label.sh: |
+    #!/bin/bash
+    while true; do
+        while [ ! -f /etc/shared/pods ]; do
+            sleep 1
+        done
+        pods=$(< "/etc/shared/pods")
+        echo "new master elected, updating label(s)..."
+        kubectl label pod --field-selector metadata.name="${pods//;*}" isMaster="true" --overwrite
+        test ! -z ${pods##*;} && kubectl label pod --field-selector metadata.name="${pods##*;}" isMaster="false" --overwrite
```

Comment:
Why are we using a single "/etc/shared/pods" files? Wasn't is simpler using "previous" and "current" files?

Reply:
> Why are we using a single "/etc/shared/pods" files? Wasn't is simpler using "previous" and "current" files?

It's due to the "init" master when we need to init (or reset) the cluster, we only need the current pod (that can lead to issue on the foreach (previous empty)).

With pods, it simplify the conf in the script to updates labels.

Reply:
Got it! Can't we assume that an existing `/etc/shared/current` file with no `/etc/shared/previous` file implies the same situation and adapt the logic based on that?

Reply:
Wouldn't it be as simple as this?
```bash
while true; do
    while [ ! -f "/etc/shared/current" ]; do
        sleep 1
    done
    echo "new master elected, updating label(s)..."
    kubectl label pod --field-selector metadata.name="$(< "/etc/shared/current")" isMaster="true" --overwrite
    if [ -f /etc/shared/previous ]; then
        kubectl label pod --field-selector metadata.name="$(< "/etc/shared/previous")" isMaster="false" --overwrite
    fi
    rm "/etc/shared/current" "/etc/shared/previous"
done
```

Reply:
@juan131 done!